src/snapred/backend/data/GroceryService.py

```diff
@@ -550,6 +550,18 @@ def fetchNeutronDataSingleUse(self, runNumber: str, useLiteMode: bool, loader: s
         if useLiteMode:
             self.convertToLiteMode(workspaceName)
 
+        config = self.dataService._readInstrumentParameters()
+        width = config["width"]
+        frequency = config["frequency"]
+        self.mantidSnapper.RemovePromptPulse(
+            "Removing prompt pulse",
+            InputWorkspace=workspaceName,
+            OutputWorkspace=workspaceName,
+            Width=width,
+            Frequency=frequency,
+        )
+        self.mantidSnapper.executeQueue()
```

Comment:
Won't this end up performing the operation on the workspace twice?  Because it's performed on the original data on load, then performed a second time on the clone.